src/main/java/com/JSONpostroute/utils/SumDigits.java

```diff
@@ -0,0 +1,11 @@
+package com.JSONpostroute.utils;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class SumDigits {
```

Comment:
In this file we are going to add more general functions like "getLetters(string text)" or another function and if we are going to use getLetters we need to call it and now we will do this "SumDigits.getLetters(...)"

**SumDigits** is not a good name for class that will get more general functions.

Reply:
Thank you for all the notes and quick replies! Please let me know if there is anything else I can refactor